.github/workflows/release.deploy.yml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2024 Ian Lewis
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: deploy
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+  workflow_call:
+    inputs:
+      workload_identity_provider:
+        description: "Google Cloud Workload Identity Provider"
+        type: string
+        required: false
+      service_account:
+        description: "Google Cloud Service Account"
+        type: string
+        required: false
+    secrets:
+      CLASP_CREDENTIALS:
+        description: "Google Apps Script credentials for clasp (legacy method)"
+        required: false
```

Comment:
Please remove the fallback method using `CLASP_CREDENTIALS` and use only workload identity federation.